### PR TITLE
Hong

### PR DESCRIPTION
**Pull request by chloe64**

### CODE REVIEW

**Comment by shiqio3o** [comment]
no bugs found

File: src/api/Search.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package api;
+
+import entity.recipe.Recipe;
+import entity.recipe.CommonRecipe;
+import entity.search_results.CommonSearchResult;
+import entity.search_results.SearchResult;
+import okhttp3.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
```
Comment (shiqio3o) [inline src/api/Search.java:10]: good

**Review by shiqio3o — APPROVED** [review]
no bugs found

**Comment by chloe64** [comment]
folders added

File: src/view/MyFolderView.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package view;
+
```
Comment (yuxizzz) [inline src/view/MyFolderView.java:2]: Need to implement in the future

**Review by yuxizzz — APPROVED** [review]
good

**Pull request merged**